src/cli/remove.rs

```diff
@@ -58,13 +66,13 @@ pub async fn execute(args: Args) -> miette::Result<()> {
     // updating prefix after removing from toml
     get_up_to_date_prefix(
```

Comment:
Shouldn't this be entirely skipped when `--no-lockfile-update` is used?

Reply:
Otherwise, I am not sure https://github.com/prefix-dev/pixi/issues/1669 will actually be fixed?